packages/ui-concerto/package.json

```diff
@@ -3,24 +3,26 @@
   "version": "0.94.0",
   "private": false,
   "dependencies": {
+    "@accordproject/concerto-core": "^0.82.8",
     "@babel/runtime": "^7.10.3",
+    "lodash.get": "^4.4.2",
```

Comment:
Are these used?

Reply:
Yes, they are all used. In `reactformvisitor.js` and `concertoform.js`. Adding the new `urijs` dependency caused the order of dependencies to change.